src/components/simple-table/SimpleTable.tsx

```diff
@@ -312,6 +346,53 @@ export const SimpleTable = <T extends object>({
       ));
     }
 
+    if (collapsible.enabled && collapsible.collapseContent?.length) {
+      return items.map((item, index) => {
+        const key = keyExtractor(item);
+        const collapsed = !expandedItemIds.includes(key);
+        return (
+          <TableRowCollapse
+            key={key}
+            className={clsx(
+              !noHoverBackground && styles.withHover,
+              collapsed && styles.collapsed,
+            )}
+            collapsed={collapsed}
+            onToggleCollapse={() => toggleItem(key)}
+            rowContent={
+              <>
+                {selectable.enabled && (
+                  <td>
+                    {selectable.renderCustomRowCheckbox?.(item, index) || (
+                      <Checkbox
+                        checked={
+                          selectable.isRowChecked?.(item, index) || false
+                        }
+                        disabled={
+                          selectable.isRowCheckboxDisabled?.(item, index) ||
+                          false
+                        }
+                        onChange={checked =>
+                          selectable.onRowCheckboxChange?.(checked, item, index)
+                        }
+                      />
+                    )}
```

Comment:
Is there a storybook way to preview selectable and collapsible?

Reply:
Oh it was my intention, at this time to have them be mutually exclusive. Similar with having onRowClick, it'll take some more work to sort out the click listeners between the different "modes". I'll add a couple quick comments about that. Those will come in a later version (probably when a design calls for them).

Reply:
Lol jk, selectable actually seems to work pretty well. I'm going to add it to the stories.

Reply:
At this point it may be worth creating a separate table as this one is far from simple now. Originally when I created it I wanted to just be able to pass the items and headers and not have to worry about rendering logic. At that point the way to build collapsible tables was to use custom `TableRowCollapse`.

Reply:
The main advantage in that case being when we make updates specific to collapsing tables or such, that we don't have to worry about breaking other existing tables that are unrelated.

Reply:
I don't know. I find the SimpleTable incredibly useful, even as it becomes a bit more "complicated." Being able to just pass some props and the rendering and styles are already taken care of is pretty great, especially after getting used to how the thing works. My hope with these changes was to be able to keep using the code patterns we have throughout Dashboard with the new designs coming out (the bordered table style).

If we want break things apart, I suppose I can start over, but I worry that it will be a bunch of extra code that is essentially the same as SimpleTable for one change in functionality.

Or maybe we just change the name of the component...

Reply:
hmmm, I see both points of view. I lean towards agreeing with Nick. I don't like the idea of having multiple amino tables that are commonly used and need to be updated separately. But I agree that any change to the simple table imposes a risk on other use cases in dashboard. I wonder if there is a better way to protect against that.

Reply:
This might be a conversation for our planning meeting. For what it's worth, everything associated with this change is optional, so it _shouldn't_ change any of the current tables in Dashboard. However, if changes are made to `TableRowCollapse`, it will affect SimpleTables using the `collapsible` functionality.